test/gtest/uct/test_peer_failure.cc

```diff
@@ -437,6 +439,10 @@ size_t test_uct_peer_failure_multiple::get_tx_queue_len() const
         tx_queue_len = 256;
         UCS_TEST_MESSAGE << name << " setting not found, "
                          << "taken test default value: " << tx_queue_len;
+        if (set) {
+            UCS_TEST_ABORT(name + " config name must be found for %s transport" +
```

Comment:
why do we need abort here?

Reply:
I think it is better to notify developers if something was changed. e.g. changed the name of the configuration attribute, but forgot to change `..._TX_QUEUE_LEN` to the new one.